opensfm/src/geometry/triangulation.h

```diff
@@ -68,7 +68,11 @@ std::pair<bool, Eigen::Matrix<T, 3, 1>> TriangulateTwoBearingsMidpointSolve(
 
   const T eps = T(1e-30);
   const T det = A.determinant();
+  #ifdef __aarch64__
+  if (std::abs<T>(det) < eps) {
```

Comment:
@pierotofy 
Thanks for your diff. Could you remove the `#ifdef` and simply use the templated version as default:

```
if (std::abs<T>(det) < eps)
```

Reply:
Hey @fabianschenk :hand: thanks for the review, I actually started out by just having the templated version, but Visual Studio doesn't like it (read: it doesn't compile) on Windows.

I could replace the define to a `#ifndef __MSVC__ `?

Reply:
Hi @pierotofy ,
Thanks for the quick response and also for testing on Windows.
Then I'll land it like it is. 

Thanks again for your contribution 👍

Reply:
Pleasure! OpenSfM is a great software.